test/sanity/phases/smoke-tests.sh

```
@@ -0,0 +1,238 @@
+    echo "FAIL" > "$RESULTS_DIR/duckdb"
+  else
+    echo "PASS" > "$RESULTS_DIR/duckdb"
+  fi
+}
+
+test_postgres() {
+  if [ -z "${TEST_PG_HOST:-}" ]; then
+    echo "SKIP" > "$RESULTS_DIR/postgres"
+    return
+  fi
+  cd "$WORKDIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)